### ISSUE TITLE
Social Icons not clickable

### ISSUE THREAD

**Issue by BookOfGreg** [joeblau/joeblau.com#1]
Tried to contact you about seeing the gitignore.io info and turns out the icons are not clickable on your current site.
Also noticed this repo is unused for now but didn't think of a better place to comment 😄

**Comment by joeblau** [comment]
Yep, I know.  Waiting for a new version of blocs app and then I'm going to re-build my site.
